documents/api/filtersets.py

```diff
@@ -55,6 +55,36 @@ def filter(self, qs, value):
         return qs
 
 
+class MultiStringFilter(Filter):
```

Comment:
Is this same thing than combination of CharFilter and BaseInFilter?

Reply:
Fixed